Force install of auxiliary files in makedist.sh

k0ekk0ek opened the pull request:
What it says on the tin.

k0ekk0ek commented:
Turns out this doesn't work as expected. I was notified the shebang was changed for the install script. RC1 shipped with `#!/usr/bin/sh` while normally that is `#!/bin/sh`. I'm assuming it has to do with me testing this on a Linux workstation(?) Something to look into for the actual 4.9.0 release.

wcawijngaards commented:
For Unbound, say 1.19.3, the release ships with install-sh with `#!/usr/bin/sh` in it. That has no troubles, and also has not changed recently.

Commit:
Use rooted temporary path in makedist.sh

Running autoconf -i in makedist.sh does not generate the required
auxiliary files if the files are available in a parent directory. Use a
temporary directory not located under the current working directory to
avoid the situation. Run configure to verify the files are really
available.

Fixes #321.

Commit:
Set up release notes for bugfix release
